index.js

```diff
@@ -0,0 +1,16 @@
+function splitNumber(number){
+    return number.toString().split('.');
+    }
+  
+  function handleHours(){
+      const inputNumber = document.getElementById("input-number");
+      const number = inputNumber.value;
+
+      let [int, decimal] = splitNumber(number);
```

Comment:
Prefira sempre const ja que voce nao vai reatribuir um valor a esses caras
```suggestion
      const [int, decimal] = splitNumber(number);
```